src/main/scala/com/itechart/ny_accidents/report/Reports.scala

```diff
@@ -10,11 +10,15 @@ class Reports {
     data.map(obj => Seq(obj._1.toString, obj._2.toString)).toSeq
   }
 
-  def generateReportStringFor3Fields[A,B,C](data: RDD[(A, B, C)]): Seq[Seq[String]] = {
+  def generateReportStringFor3Fields[A, B, C](data: RDD[(A, B, C)]): Seq[Seq[String]] = {
     data.map(obj => Seq(obj._1.toString, obj._2.toString, obj._3.toString)).collect().toSeq
   }
 
-  def generateReportStringFor2Fields[A,B](data: RDD[(A, B)]): Seq[Seq[String]] = {
+  def generateReportStringFor4Fields[A, B, C, D](data: RDD[(A, B, C, D)]): Seq[Seq[String]] = {
+    data.map(obj => Seq(obj._1.toString, obj._2.toString, obj._3.toString, obj._4.toString)).collect().toSeq
```

Comment:
Please, refactor this piece of code using pattern matching

Reply:
In this piece of code we don't know what data comes to us, and we just use toString method to each tuple variable. So there is no need in good names. 
Also these methods will be rewriting in next sprint

Reply:
Ok, thanks